samples/advanced-markers-accessibility/index.ts

```diff
@@ -54,7 +53,7 @@ async function initMap() {
         // [START maps_advanced_markers_accessibility_marker]
         const marker = new AdvancedMarkerElement({
             position,
-            map,
+            map: innerMap,
```

Comment:
Let's not use innerMap, let's simply do this:

```
mapElement.append(marker);
```

No need to pass the map property :)